### ISSUE TITLE
Issue with CRUD generation

### ISSUE THREAD

**Issue by abhinavlal** [BRACKETS-by-TRIAD/craftable-pro-docs#84]
Getting following error during CRUD generation

```
TypeError

  Brackets\CraftablePro\Commands\CrudGeneratorCommand::buildRule(): Argument #1 ($columnType) must be of type string, null given, called in ~/vendor/brackets/craftable-pro/src/Commands/Trait/BuildCrudGeneratorTrait.php on line 25

  at vendor/brackets/craftable-pro/src/Commands/Trait/BuildCrudGeneratorTrait.php:366
    362▕
    363▕     /**
    364▕      * Return rules for requests.
    365▕      */
  ➜ 366▕     protected function buildRule(string $columnType, bool $required): array
    367▕     {
    368▕         $rules = collect([]);
    369▕         $rules->push($required ? 'required' : 'nullable');
    370▕

      +19 vendor frames

  20  artisan:13
      Illuminate\Foundation\Application::handleCommand(Object(Symfony\Component\Console\Input\ArgvInput))
```

Model for which i am trying to generate CRUD for
```php
 Schema::create('customer', function (Blueprint $table) {
            $table->id();
            $table->timestamps();
            $table->string('facility_name');
            $table->string('email');
            $table->string('phone');
            $table->enum('currency', ['INR', 'USD', 'AED', 'SAR']);
            $table->string('first_name');
            $table->string('last_name');
        });
```

Laravel version - 11.0
brackets/craftable-pro - 2.0

**Comment by palypster** [comment]
Thank you for the report. We will get back to you tomorrow.

**Comment by strstensky** [comment]
Hello, which type of database are you using: PostgreSQL or MySQL?

**Comment by abhinavlal** [comment]
Mysql version - 8.0.33

**Comment by strstensky** [comment]
The issue has been fixed in release v2.0.2 (see https://docs.craftable.pro/changelog#v202---2024-04-15).
